[12.0] [MIG] sale_operating_unit (#151)
OCA/operating-unit: sale_operating_unit/readme/DESCRIPTION.rst

> This module was written to extend the Sales capabilities of Odoo.

Reviewer: ```suggestion
This module extends the Sales capabilities of Odoo
```

> This module introduces the operating unit to the Sales Order.

Reviewer: ```suggestion
and  introduces the operating unit to the Sales Order object.
```

Security rules are defined to ensure that users can only display the

> Sales Orders in which they are allowed access to.

Reviewer: ```suggestion
Sales Orders they are allowed to access.
```
